src/xma/include/lib/xmaxclbin.h

```diff
@@ -74,7 +74,7 @@ typedef struct XmaXclbinInfo
     uint32_t    reserved[32];
 } XmaXclbinInfo;
 
-char *xma_xclbin_file_open(const char *xclbin_name);
+std::vector<char> xma_xclbin_file_open(const char *xclbin_name);
 int xma_xclbin_info_get(char *buffer, XmaXclbinInfo *info);
```

Comment:
'const char* buffer' to make it clear that this function is not changing the buffer content.

Reply:
this change is coming in separate pr for these other functions